tools/telemetry-generator/src/commands/appInsights.ts

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fs from "fs";
+import path from "path";
+import { Command, Flags } from "@oclif/core";
+import * as appInsight from 'applicationinsights';
+
+export class EntryPoint extends Command {
+    static flags = {
+        help: Flags.help(),
+        handlerModule: Flags.string({
+            char: "m",
+            required: true,
+            description: "Absolute path to a JavaScript file that exports a handler function to process the files " +
+                "contained in the folders specified with --dir.",
+        }),
+        dir: Flags.string({
+            char: "d",
+            multiple: true,
+            required: true,
+            description: "Folder that contain the test output files to process. " +
+                "Files in subfolders are also processed. Can be specified multiple times.",
+        }),
+        connectionString: Flags.string({
+            char: "c",
+            required: true,
+            description: "The connection string to initialize the azure telemetry client with",
+        }),
+    };
+
+    static examples = [
+        {
+            command: "$ node bin/run --handlerModule /path/to/my/module.js --dir /path/to/my/files",
+            description: "Process files from /path/to/my/files and all its subfolders, using the handler at " +
+                "/path/to/my/module.js",
+        },
+    ];
```

Comment:
Update this with the extra pieces for the appInsights command.

Reply:
Done in recent commit # 84045